docs/Reference/JSON-RPC-API-Methods.md

```diff
@@ -0,0 +1,1911 @@
+    connected (press CTRL+C to quit)
+    > {"jsonrpc":"2.0","method":"eth_gasPrice","params":[],"id":53}
+    ```
+    
+    ```json tab="JSON result"
+    {
+      "jsonrpc" : "2.0",
+      "id" : 53,
+      "result" : "0x3e8"
+    }
+    ```
+
+### eth_accounts
+
+Returns a list of account addresses that the client owns.
+
+!!!note
+    This method returns an empty object because Pantheon [does not support account management](Using-JSON-RPC-API.md#account-management-not-supported-by-pantheon).
+
+*Parameters*
+
+None
+
+*Returns*
+
+`Array of data` : List of 20-byte account addresses owned by the client.
+
+!!! example
+    ```bash tab="curl HTTP request"
+    $ curl -X POST --data '{"jsonrpc":"2.0","method":"eth_accounts","params":[],"id":53}' <JSON-RPC-http-endpoint:port>
+    ```
+    
+    ```bash tab="wscat WS request"
+    $ wscat -c <JSON-RPC-ws-endpoint:port>
+    connected (press CTRL+C to quit)
+    > {"jsonrpc":"2.0","method":"eth_accounts","params":[],"id":53}
+    ```
+    
+    ```json tab="JSON result"
+    {
+      "jsonrpc" : "2.0",
+      "id" : 53,
+      "result" : [ ]
+    }
+    ```
+
+### eth_blockNumber
+
+Returns the index of the current block the client is processing.
+
+*Parameters*
+
+None
+
+*Returns*
+
+`result` : *QUANTITY* - Hexadecimal integer representing the 0-based index of the block that the client is currently processing.
+
+
+!!! example
+    ```bash tab="curl HTTP request"
+    $ curl -X POST --data '{"jsonrpc":"2.0","method":"eth_blockNumber","params":[],"id":51}' <JSON-RPC-http-endpoint:port>
+    ```
+    
+    ```bash tab="wscat WS request"
+    $ wscat -c <JSON-RPC-ws-endpoint:port>
+    connected (press CTRL+C to quit)
+    > {"jsonrpc":"2.0","method":"eth_blockNumber","params":[],"id":51}
+    ```
+    
+    ```json tab="JSON result"
+    {
+      "jsonrpc" : "2.0",
+      "id" : 51,
+      "result" : "0x2377"
+    }
+    ```
+
+### eth_getBalance
+
+Returns the account balance of the specified address.
+
+*Parameters*
+
+`DATA` - 20-byte account address from which to retrieve the balance.
+
+`QUANTITY|TAG` - Integer representing a block number or one of the string tags `latest`, `earliest`, or `pending`, as described in [Block Parameter](Using-JSON-RPC-API.md#block-parameter).
+
+*Returns*
+
+`result` : *QUANTITY* - Integer value of the current balance in wei.
+
+!!! example
+    ```bash tab="curl HTTP request"
+    $ curl -X POST --data '{"jsonrpc":"2.0","method":"eth_getBalance","params":["0xdd37f65db31c107f773e82a4f85c693058fef7a9", "latest"],"id":53}' <JSON-RPC-http-endpoint:port>
+    ```
+    
+    ```bash tab="wscat WS request"
+    $ wscat -c <JSON-RPC-ws-endpoint:port>
+    connected (press CTRL+C to quit)
+    > {"jsonrpc":"2.0","method":"eth_getBalance","params":["0xdd37f65db31c107f773e82a4f85c693058fef7a9", "latest"],"id":53}
+    ```
+    
+    ```json tab="JSON result"
+    {
+      "jsonrpc" : "2.0",
+      "id" : 53,
+      "result" : "0x0"
+    }
+    ```
+
+### eth_getStorageAt
```

Comment:
There's a paragraph mark after this heading in the output?

Reply:
Actually - I've just realised that's the perma link which is a good feature.  Can we change the symbol it uses to the be the link symbol?  Or is this out of our control?

Reply:
Fixed with fontawesome web font https://fontawesome.com/icons/link?style=solid